src/main/java/org/gridsuite/explore/server/services/DirectoryService.java

```diff
@@ -129,7 +139,7 @@ public void setDirectoryServerBaseUri(String directoryServerBaseUri) {
     }
 
     public Flux<ElementAttributes> listDirectoryContent(UUID directoryUuid, String userId) {
-        String path = UriComponentsBuilder.fromPath(DELIMITER + DIRECTORY_SERVER_API_VERSION + "/directories/{directoryUuid}/content")
+        String path = UriComponentsBuilder.fromPath(DELIMITER + DIRECTORY_SERVER_API_VERSION + "/directories/{directoryUuid}/elements")
```

Comment:
You can use `DIRECTORIES_SERVER_ROOT_PATH` instead of `DELIMITER + DIRECTORY_SERVER_API_VERSION + "/directories`

Reply:
Done